osquery/database/plugins/rocksdb.cpp

```diff
@@ -243,6 +259,28 @@ void RocksDBDatabasePlugin::close() {
   }
 }
 
+void RocksDBDatabasePlugin::repairDB() {
+#if defined(ROCKSDB_LITE)
+  // ROCKSDB_LITE does not have a RepairDB method. No option but to delete the
+  // corrupted DB
+  LOG(INFO) << "repairDB: Deleting corrupted db files";
+  std::vector<std::string> file_names;
+  auto s = listFilesInDirectory(path_, file_names);
+  if (s.ok()) {
+    for (auto file : file_names) {
+      fs::path file_path(file);
+      osquery::remove(file_path);
+    }
+  } else {
+    LOG(INFO) << "Unable to list " << path_ << ". Error " << s.toString();
+  }
+#else
+  auto rs = rocksdb::RepairDB(path_, options_, column_families_);
```

Comment:
Let's ping the RocksDB project and find out if this can be included in the LITE version.